bot/exts/filtering/_settings_types/validations/channel_scope.py

```diff
@@ -31,10 +31,10 @@ class ChannelScope(ValidationEntry):
 
     # NOTE: Don't change this to use the new 3.10 union syntax unless you ensure Pydantic type validation and coercion
     # work properly. At the time of writing this code there's a difference.
-    disabled_channels: set[Union[int, str]]
-    disabled_categories: set[Union[int, str]]
-    enabled_channels: set[Union[int, str]]
-    enabled_categories: set[Union[int, str]]
+    disabled_channels: set[int | str]
+    disabled_categories: set[int | str]
+    enabled_channels: set[int | str]
+    enabled_categories: set[int | str]
```

Comment:
🥲

Reply:
This is part of the [pyupgrade](https://beta.ruff.rs/docs/rules/#pyupgrade-up) rule, specifically `UP007`. Personally I think having one consistent way here is better, so choosing the less verbose one, that doesn't require an import is the way I'd go too.

Reply:
Sure, but did you read the comment above :D

Reply:
Reverted in https://github.com/python-discord/bot/pull/2523/commits/65ec4bfe0404d6081e377c274cd34274e7eeb0db